crypto/sign_norelic.go

```diff
@@ -1,3 +1,4 @@
+//go:build !relic
 // +build !relic
```

Comment:
Not sure why both are needed. But my IDE(goland) has a problem with both while just `// +build !relic` works as expected.

Reply:
Hmm it seems like this was automatically added by VSCode.